game/templates/game/landing.html

```
@@ -237,6 +237,39 @@ <h3 class="card-title">Hybrid Architecture</h3>
+                <a href="{% url 'rules' %}" class="footer-link">Rulebook</a>
+                <a href="{% url 'stats' %}" class="footer-link">Statistics</a>
+                <a href="{% url 'index' %}" class="footer-link">Play Game</a>
+            </div>
+            <div class="footer-col">
+                <h4>COMMUNITY</h4>
+                <a href="https://github.com/Checkora" target="_blank" rel="noopener noreferrer" class="footer-link">GitHub Organization</a>
+                <div style="color: var(--text-secondary); font-size: 0.95rem; font-weight: 500;">Maintainers: <a href="https://github.com/triemerge" target="_blank" rel="noopener noreferrer" class="footer-link">@triemerge</a> &amp; <a href="https://github.com/EDWARD-012" target="_blank" rel="noopener noreferrer" class="footer-link">@EDWARD-012</a></div>
+                <a href="https://discord.gg/WfrpMuNZn" target="_blank" rel="noopener noreferrer" class="footer-link">Discord Community</a>
+            </div>
+        </div>
+        <div class="footer-bottom">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div> ], start tag match failed [ <div class="footer-bottom"> ] on line 262.

(tag-pair)